modules/nixos/remote-builder/build04.nix

```diff
@@ -9,7 +9,7 @@
       sshKey = config.sops.secrets.id_buildfarm.path;
       sshUser = "nix";
       system = "aarch64-linux";
-      supportedFeatures = [ "big-parallel" ]; # sync with build04/configuration.nix
+      supportedFeatures = inputs.self.nixosConfigurations.build04.config.nix.settings.system-features;
```

Comment:
What's the impact on evaluation speed of this?

Reply:
Not much but I guess we may as well avoid it where we can, reverted this change.